npm-debug.log

```diff
@@ -0,0 +1,19 @@
+0 info it worked if it ends with ok
```

Comment:
don't include logs like this in commits. not sure why npm would be involved in building Hazel, but if you're doing something locally then you can add these kinds of files to .gitignore.